doc/cephadm/compatibility.rst

```diff
@@ -28,27 +27,32 @@ Those versions are expected to work:
 +-----------+-------+-------+-------+-------+-------+
 
 .. warning:: 
-  Only podman versions that are 2.0.0 and higher work with Ceph Pacific, with the exception of podman version 2.2.1, which does not work with Ceph Pacific. kubic stable is known to work with Ceph Pacific, but it must be run with a newer kernel.
+
+   To use Podman with Ceph Pacific, you must use **a version of Podman that
+   is 2.0.0 or higher**. However, **Podman version 2.2.1 does not work with
+   Ceph Pacific**.
+   
+   "Kubic stable" is known to work with Ceph Pacific, but it must be run
+   with a newer kernel.
```

Comment:
@zdover23 I won't delay this PR, but I will note that "newer kernel" could stand to be more precise. Newer than what, exactly?